sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.1.0-beta.1 (Unreleased)
```

Comment:
```suggestion
## 12.0.1 (2021-06-09)
```

Reply:
Are we shipping a patch hot-fix?

Reply:
Good point, perhaps this should be "12.1.0".  I would NOT call this a beta though.

Reply:
@RickWinter The version is auto-generated by the release pipeline. We can fix it when shipping next release.

It's not related to this PR anyway.

Reply:
I agree, let's leave the version as is, until we are actually read to ship.